.buildkite/cpu_scaling_pipeline/pipeline.yml

```
@@ -0,0 +1,548 @@
+agents:
+  queue: new-central
+  slurm_mem: 8G
+  modules: climacommon/2024_12_16
+
+env:
+  JULIA_LOAD_PATH: "${JULIA_LOAD_PATH}:${BUILDKITE_BUILD_CHECKOUT_PATH}/.buildkite"
+  OPENBLAS_NUM_THREADS: 1
+  SLURM_KILL_BAD_EXIT: 1
+  JULIA_NVTX_CALLBACKS: gc
+  JULIA_MAX_NUM_PRECOMPILE_FILES: 100
+  JULIA_DEPOT_PATH: "${BUILDKITE_BUILD_PATH}/${BUILDKITE_PIPELINE_SLUG}/depot/default"
+  CONFIG_PATH: "config/model_configs"
+  MBW_SCALING_CONFIG_PATH: "config/mbw_scaling_configs"
+  CPU_TYPE: "icelake"
+
+steps:
+  - label: "init :computer:"
+    key: "init_cpu_env"
+    command:
+      - "echo $$JULIA_DEPOT_PATH"
+
+      - echo "--- Instantiate project"
+      - "julia --project -e 'using Pkg; Pkg.instantiate(;verbose=true)'"
+      - "julia --project -e 'using Pkg; Pkg.precompile()'"
+      - "julia --project -e 'using Pkg; Pkg.status()'"
+
+      - echo "--- Instantiate examples"
+      - "julia --project=examples -e 'using Pkg; Pkg.instantiate(;verbose=true)'"
+      - "julia --project=examples -e 'using Pkg; Pkg.precompile()'"
+      - "julia --project=examples -e 'using Pkg; Pkg.status()'"
+
+    agents:
+      slurm_cpus_per_task: 8
+    env:
+      JULIA_NUM_PRECOMPILE_TASKS: 8
+      JULIA_MAX_NUM_PRECOMPILE_FILES: 50
+
+  - wait
+
+  - group: "Moist Baroclinic Wave, helems = 30, strong scaling"
+    steps:
+
+      - label: ":computer: 1 node with 16 processes per node"
+        command: >
+          srun julia --color=yes --project=examples examples/hybrid/driver.jl
+          --config_file $MBW_SCALING_CONFIG_PATH/moist_baroclinic_wave_helem_30_0M_ss.yml
+          --job_id moist_baroclinic_wave_helem_30_0M_ss_16process
+
+        artifact_paths: "moist_baroclinic_wave_helem_30_0M_ss_16process/output_active/*"
+        agents:
+          slurm_constraint: $CPU_TYPE
+          queue: new-central
+          slurm_nodes: 1
+          slurm_tasks_per_node: 16
+          slurm_cpus_per_task: 1
+          slurm_mem: 0
+          slurm_time: 1:00:00
+          slurm_exclusive:
```

```suggestion
          slurm_exclusive: true
          slurm_reservation: false
```
Two notes:
`slurm_exclusive` needs to be set to true. When there is no key, buildkite drops the key. This shouldn't matter here since `slurm_mem: 0` effectively requires an exclusive node

Could you add `slurm_reservation: false` to all of the jobs? Otherwise they will probably take up our reserved node due to the memory and exclusive flag